style.css

```diff
@@ -2,7 +2,7 @@ body {
 	margin:0;
 	padding:0;
 	background-color:white;
-	font: 9pt arial;
+	font: 9pt 'Helvetica Neue','Nimbus Sans','Helvetica','Arial',sans-serif;
```

Comment:
Just curious: Is this a recommended font selection?

Reply:
I took this from OOUI / Wikipedia's sans-serif fonts for form elements, beats simply declaring "Arial", especially because that does not even specify whether or not it's serif or sans-serif.